modules/nw-ovn-kubernetes-session-affinity.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
+
+:_content-type: CONCEPT
+[id="nw-ovn-kubernetes-session-affinity_{context}"]
+= Session affinity
+Session affinity is a feature that applies to kubernetes objects of type `services`. You can use `session affinity` if you want to ensure that each time you connect to a ServiceVIP:Port the traffic is always load balanced to the same backend. For more information, including how to set session affinity based on a client's IP address, refer to link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity[Session affinity].
```

Comment:
s/kubernetes/Kubernetes
s/kubernetes objects of type `services`/kubernetes `services` objects (minimalism)
Does `session affinity`need markup?
Does _ServiceVIP:Port_ need a comma? _ServiceVIP:Port, the traffic..._ (I tend to over-comma)